src/services/armService.ts

```diff
@@ -190,4 +201,13 @@ export class ArmService extends BaseService {
       });
     }
   }
+
+  private getNodeVersion(input: string): string{ 
```

Comment:
Be more explicit about the purpose of this function. Something like `validateNodeVersion`